src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -2,16 +2,59 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Security.AccessControl;
 using Xunit;
 
 namespace System.IO
 {
     public class FileSystemAclExtensionsTests
     {
         [Fact]
-        public void GetAccessControl_InvalidArguments()
+        public void GetAccessControl_DirectoryInfo_InvalidArguments()
         {
             Assert.Throws<NullReferenceException>(() => FileSystemAclExtensions.GetAccessControl((DirectoryInfo)null));
         }
+
+        [Fact]
+        public void GetAccessControl_DirectoryInfo_AccessControlSections_InvalidArguments()
+        {
+            Assert.Throws<NullReferenceException>(() => FileSystemAclExtensions.GetAccessControl((DirectoryInfo)null, new AccessControlSections()));
```

Comment:
Sorry, I meant that as a general question; should we consider that this throws NRE rather than ANE to be a bug, open an issue to track it, and then change both the code in `FileSystemAclExtensions` and the tests that hit it.
As a test of the current code, this is fine.